### PR TITLE
[BLCR-3047] Updated img resizer

### PR DESCRIPTION
**Pull request by iFurt**
https://brandslisten.atlassian.net/browse/BLCR-3047

### CODE REVIEW

File: vendor/assets/javascripts/trumbowyg/vendor/plugins/resizimg/trumbowyg.resizimg.custom.js [hunk 1]
```diff
@@ -56,6 +56,16 @@
                         return ev.preventDefault();
                     };
 
+                    var setImgMeta = function(img) {
+                      if (!img[0]) return;
```
Comment (Xtimesx) [inline vendor/assets/javascripts/trumbowyg/vendor/plugins/resizimg/trumbowyg.resizimg.custom.js]: correct me when I'm wrong, but can't there be more  that 1 image in the editor?
so, shouldn't we iterate over all, not only one?

Comment (iFurt) [reply vendor/assets/javascripts/trumbowyg/vendor/plugins/resizimg/trumbowyg.resizimg.custom.js]: Yeap, you are right. Will update the function

File: vendor/assets/javascripts/trumbowyg/vendor/plugins/resizimg/trumbowyg.resizimg.custom.js [hunk 2]
```diff
@@ -56,13 +56,16 @@
...
-                      $("<img/>", {
-                        load: function(){
-                          img.css({ "min-width": this.width });
-                        },
-                        src: img[0].src
+                    var setImgMeta = function(imgs) {
+                      if (imgs.length == 0) return;
+
+                      imgs.each(function(_, img) {
+                        $("<img/>", {
```
Comment (Xtimesx) [inline vendor/assets/javascripts/trumbowyg/vendor/plugins/resizimg/trumbowyg.resizimg.custom.js]: doesn't this create a new image?
shouldn't we attach a load callback to the existing element?

and maybe only once?

**Comment by Xtimesx** [comment]
can you also do this?
Remove setTimeout there trumbowyg.resizimg.custom.js#L122
otherwise it is possible, that an image is already loaded and the callback won't be triggered

**Review by Xtimesx — APPROVED** [review]

**Pull request merged**